inginious/frontend/plugins/grader_generator/pages/grader.py

```diff
@@ -3,139 +3,71 @@
 import os
 from inginious.frontend.pages.course_admin.task_edit import CourseEditTask
 from collections import OrderedDict
-
+import re
```

Comment:
re, CourseEditTask and and OrderedDict are unused imports.

Reply:
Yes, I pass all this functionality to other file, I forgot to delete them